Test/TestCases/A/hello.js

```diff
@@ -1,7 +1,9 @@
 const assert = require('assert');
 
-// Load the addon module.
-const test = require(process.env['TEST_NODE_API_MODULE_PATH']);
+// Load the addon module, using either hosted or native AOT mode.
+const dotnetModule = process.env['TEST_DOTNET_MODULE_PATH'];
+const dotnetHost = process.env['TEST_DOTNET_HOST_PATH'];
+const test = dotnetHost ? require(dotnetHost).require(dotnetModule) : require(dotnetModule);
```

Comment:
>require(dotnetHost) [](http://example.com/codeflow?start=26&length=19)

I wonder if we can use similar technique to define multiple C# Node-API modules per DLL.
E.g.
   require('DLL path').load_module('module1')
then in another JS file:
   require('DLL path').load_module('module2')

Reply:
That would be easy to support in CLR hosting mode. We chatted about this. It's definitely possible, but would add complexity to module initialization and I'm not convinced it's needed. We can reconsider later.